migrations/1528395714_user_invalidate_session.down.sql

```diff
@@ -0,0 +1,13 @@
+BEGIN;
+
+-- Insert migration here. See README.md. Highlights:
+--  * Always use IF EXISTS. eg: DROP TABLE IF EXISTS global_dep_private;
+--  * All migrations must be backward-compatible. Old versions of Sourcegraph
+--    need to be able to read/write post migration.
+--  * Historically we advised against transactions since we thought the
+--    migrate library handled it. However, it does not! /facepalm
+DROP TRIGGER IF EXISTS trig_invalidate_session_on_password_change_or_reset ON users;
+DROP FUNCTION IF EXISTS invalidate_session_for_userid_on_password_change_or_reset();
```

Comment:
Something I just [learned from @efritz](https://github.com/sourcegraph/sourcegraph/pull/13653/files/cf3cb14c38ea05e358a459cc238ff272ad8256f2#diff-d1b54aa86b06b8df8c20e16e634bf81bR3):
```suggestion
DROP FUNCTION IF EXISTS invalidate_session_for_userid_on_password_change_or_reset() CASCADE;
```

Reply:
Good call, thanks!